images/frankenphp/Caddyfile

```diff
@@ -1,4 +1,6 @@
 {
+    cache
```

Comment:
I would try to keep this file same as https://github.com/dunglas/frankenphp/blob/main/caddy/frankenphp/Caddyfile

You could set environment variable `CADDY_GLOBAL_OPTIONS` to `cache` right?